addons/mail/models/mail_channel.py

```diff
@@ -826,8 +817,7 @@ def channel_info(self):
 
             # add RTC sessions info
             info.update({
-                'invitedGuests': [('insert', [{'id': member.guest_id.id, 'name': member.guest_id.name} for member in invited_members_by_channel[channel] if member.guest_id])],
-                'invitedPartners': [('insert', [{'id': member.partner_id.id, 'name': member.partner_id.name} for member in invited_members_by_channel[channel] if member.partner_id])],
+                'channel': [('insert', {'id': channel.id, 'invitedMembers': [('insert', invited_members_by_channel[channel].mail_channel_member_format())]})],
```

Comment:
`invitedMembers` is still on thread for now, not on channel?

Reply:
or is that the wrongly named channel that is actually thread?
but if that is the case, `model` should be specified together with `id`